lib/src/drawings.dart

```diff
@@ -408,13 +440,39 @@ class IconDrawing extends GYWDrawing {
     this.scale = 1.0,
   }) : icon = null;
 
+  // TODO(roscale): make this class immutable with Freezed and use copyWith.
```

Comment:
I planned to make a PR with this immutable feature. It can be useful for the translate extension or the dark mode to avoid copying every parameters in these methods.
What does Freezed brings ? You can simply use the `@immutable` annotation, no ?